addons/sys_intercom/fnc_getIntercomConnectionStatus.sqf

```diff
@@ -0,0 +1,50 @@
+/*
+ * Author: ACRE2Team
+ * Gets the intercom connection status of the vehicle seat the unit is in.
+ *
+ * Arguments:
+ * 0: Vehicle with intercom <OBJECT>
+ * 1: Unit to be checked <OBJECT>
+ *
+ * Return Value:
+ * Intercom connection status <NUMBER>
+ *  0: no intercom
```

Comment:
Style with `- ` maybe, this looks like it's an array (yes I know it says `NUMBER`).